### DISCUSSION TITLE
Error using [q

### DISCUSSION THREAD

**Discussion by mapringg** [LazyVim/LazyVim#437] [Q&A] [answered]
<img width="1008" alt="Screenshot 2566-03-15 at 22 43 19" src="https://user-images.githubusercontent.com/15067237/225364272-609b9722-e735-47d0-9356-f04842141186.png">

When I press [q], I always get this error. As I could never get the key to function, I am not what it is intended for.

**Comment by folke** [comment, marked as answer]
That's because your quickfix list is empty. But should show a better error for sure

**Reply by folke** [reply]
It's actually normal neovim behavior. You'll get the same error, but without the stack trace when doing `:cprev`

**Reply by mapringg** [reply]
Thank you for explaining.
